src/main/java/ca/uhn/fhir/jpa/starter/interceptor/LoggerInterceptor.java

```diff
@@ -15,32 +12,56 @@
 import ca.uhn.fhir.rest.api.server.RequestDetails;
 import ca.uhn.fhir.rest.api.server.ResponseDetails;
 import ca.uhn.fhir.rest.server.servlet.ServletRequestDetails;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
 
 @Interceptor
-public class CorrelationIdInterceptor {
+public class LoggerInterceptor {
```

Comment:
See my comment below re: configuration file change.

This simple rename change needs to be made to many other repos after this.

Reply:
So, should I keep the name as it is? or change in other repos as you mentioned?

Reply:
You can keep the current name. I created an issue on the purple team board to update the different server configurations to use the new name.